alerta/settings.py

```diff
@@ -101,6 +101,10 @@
 LDAP_DOMAINS_BASEDN = {}  # type: Dict[str, str]
 LDAP_ALLOW_SELF_SIGNED_CERT = False
 
+# LDAPS
+LDAPS = False #LDAPS True/False
+LDAPS_CERT = '' #Path to certificate
```

Comment:
Default the `LDAPS_CERT` to `None` and remove the `LDAPS` option .
```
LDAPS_CERT = None  # Path to certificate
```

Then in `basic_ldap.py` it becomes...

```
    if current_app.config['LDAPS_CERT']:
        ldap.set_option(ldap.OPT_X_TLS_REQUIRE_CERT, ldap.OPT_X_TLS_HARD)
        ldap.set_option(ldap.OPT_X_TLS_CACERTFILE, current_app.config['LDAPS_CERT'])
```